foodnet/membership/models.py

```diff
@@ -63,12 +93,54 @@ class DivisionMember(models.Model):
     start = models.DateTimeField(auto_now_add=True)
     exit = models.DateTimeField()
     active = models.BooleanField(default=True)
-    
+
     class Meta:
         unique_together = (('member', 'division'),)
-    
 
-@receiver(post_save, sender=User)
+
+class Invitation(models.Model):
+    email = models.EmailField(null=False, blank=False, unique=False)
+    accepted = models.BooleanField(default=False)
+    accepted_dt = models.DateTimeField(null=True)
+    verification_key = models.UUIDField(default=uuid.uuid4, editable=False,
+                                        null=False, blank=False, unique=True)
+    created = models.DateTimeField(auto_now_add=True)
+    invited_by = models.ForeignKey(User)
+    division = models.ForeignKey(Division)
+    member_category = models.ForeignKey(MemberCategory)
+
+    class Meta:
+        permissions = (
+            ("can_invite", "Can send invitation"),
+        )
+
+    def __str__(self):
+        if self.accepted:
+            acp = 'accepted'
+        else:
+            acp = 'NOT accepted'
+        return '{} {}>'.format(self.email, acp)
+
+
+@receiver(post_save, sender=User, dispatch_uid='membership-user-profile')
 def create_user_profile(sender, instance, created, **kwargs):
     if created:
         UserProfile.objects.create(user=instance)
+
+
+@receiver(post_save, sender=Invitation,
+          dispatch_uid='membership-invitation-send_email_invitation')
+def send_email_invitation(sender, instance, created, **kwargs):
+    if created:
+        subject = 'You have been invitate to FoodNet!'
+        to_addrs = [instance.email, ]
+        invite_url = absolute_url_reverse('accept_invitation', kwargs=dict(
+                                 verification_key=
+                                 instance.verification_key.hex))
+        body = """
```

Comment:
We should get this out as an template, and maybe look into using `paloma` (https://github.com/iconfinder/paloma) for emails.